charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -100,8 +118,14 @@ spec:
                 - name: interserver
                   containerPort: 9009
               volumeMounts:
+              {{- if (eq (include "clickhouse.persistentVolumeClaim" . ) "" ) }}
                 - name: data-volumeclaim-template
```

Comment:
I wonder if we can always use `clickhouse-storage` name and simplify by dropping this condition here (& renaming on line 145), but I don't know what the name change would do for upgrades 😅 so ignore this.